### ISSUE TITLE
logout process miss some actions

### ISSUE THREAD

**Issue by QuentinGruber** [QuentinGruber/h1z1-server#888]
Like login out a server and join another could lead to multiple issues like speed 0 and some loadout interactions being gone.

**Comment by QuentinGruber** [comment]
Linked to #727

**Comment by sofresh007** [comment]
Also if you die in a vehicle you get the same error..

**Comment by QuentinGruber** [comment]
> Also if you die in a vehicle you get the same error..

Yep this issue is currently solved on https://github.com/QuentinGruber/h1z1-server/pull/875 , was some client side glitch linked to a bad sendself or order of packets
